src/ElectricPokemon.java

```diff
@@ -0,0 +1,73 @@
+public class ElectricPokemon extends Pokemon {
```

Comment:
Je out-map hoef je niet op github te zetten. Die kun je in je .gitignore zetten, samen met je .idea-map